.prettierrc

```diff
@@ -1,9 +1,3 @@
 {
-  "printWidth": 200,
-  "tabWidth": 2,
-  "useTabs": false,
-  "semi": true,
-  "singleQuote": true,
-  "trailingComma": "none",
-  "proseWrap": "preserve"
-}
+  "extends": "node_modules/playkit-prettier-config/prettier.config"
```

Comment:
we should also share lint settings like here https://engineering.invisionapp.com/post/sharing-eslint-across-teams/.
that way if some lint rules conflict with prettier right now, we can fix them in one place.